container/health.go

```diff
@@ -19,12 +19,7 @@ func (s *Health) String() string {
 		return types.Unhealthy
 	}
 
-	switch s.Status {
-	case types.Starting:
-		return "health: starting"
```

Comment:
If you take a look at `daemon/health.go` specifically 

```
func (d *Daemon) initHealthMonitor(c *container.Container) {
	// If no healthcheck is setup then don't init the monitor
	if getProbe(c) == nil {
		return
	}

	// This is needed in case we're auto-restarting
	d.stopHealthchecks(c)

	if c.State.Health == nil {
		h := &container.Health{}
		h.Status = types.Starting
		c.State.Health = h
	}

	d.updateHealthMonitor(c)
}
```

You can see the status for a starting container gets set to the type.Starting enum.